docs/manuals/ZAC-gebruikershandleiding/ZAC-gebruikershandleiding.md

```diff
@@ -0,0 +1,1470 @@
+#
+![Dimpact](./images/pic.svg)
+
+**_Zaakafhandelcomponent (ZAC) gebruikershandleiding_**
+
+**_ZAC_**
+
+|     |     |     |
+| --- | --- | --- |
+|     |     | Colofon |
+| Datum | :   | 29-8-2023 |
+| Versie | :   | 1.5.1 |
+| Verandering | :   | Verduidelijking zoekfunctie |
+| Project referentie | :   | ZAC |
+| Toegangsrechten | :   | Alleen lezen |
+| Status | :   | Definitief |
+| Redacteur | :   | Karin Masselink |
+| Auteur(s) | :   | Roy Buis, Camiel Braun |
+|     |     |     |
+
+Versiegeschiedenis:
+
+| 0.1 | Initiële versie |
+| --- | --- |
+| 0.2 | Update nav sprints opgeleverd op 30-11-2022, 14-12-2022, 11-1-2023, 25-1-2023 |
+| 1.0 | Update nav sprints opgeleverd op 9-2-2023 en 23-2-2023<br><br>Heropenen zaak, Zaak koppelen, Werking van de taak: Advies extern toegevoegd |
+| 1.1 | Update nav sprints opgeleverd op 8-3-2023 en 22-3-2023 |
+| 1.2 | Update nav sprints opgeleverd op 5-4-2023 en 19-4-2023 |
+| 1.3 | Update nav sprints opgeleverd op 3-5-2023 en 31-5-2023 |
+| 1.4 | Update format wijziging en redigeerwerk van 29-8-2023 |
+| 1.5 | Update nav sprints opgeleverd van 10-2023 tot 27-05-2024 |
+| 1.5.1 | Update zoekfunctie bedrijf en rechtspersoon |
```

Comment:
Ik zou zeggen dat dit wel een nieuwe versie waard is toch? Ook al is er inhoudelijk niks veranderd.

```suggestion
| 1.5.1 | Update zoekfunctie bedrijf en rechtspersoon |
| 1.6.0 | Gebruikershandleiding geconverteerd naar Markdown |
```

Reply:
Daarom, had ik inderdaad de versie gelijk gehouden. Maar mag dit eigenlijk niet helemaal weg? Nu houd git toch de versies bij?